package.json

```diff
@@ -7,16 +7,16 @@
   "license": "MIT",
   "scripts": {
     "build": "run-s -n build:chrome build:firefox build:safari",
-    "build:chrome": "cross-env VARIANT=chrome webpack --mode=production --config webpack.webext.babel.js",
-    "build:firefox": "cross-env VARIANT=firefox webpack --mode=production --config webpack.webext.babel.js",
+    "build:chrome": "cross-env VARIANT=chrome webpack --config webpack.webext.babel.js",
```

Comment:
Hm, I honestly don't understand how this makes a difference 🙈 

According to [the docs](https://webpack.js.org/configuration/mode/):
> If not set, webpack sets production as the default value for mode.

Reading this, I'd have expected the build to be 1:1 the same 🤔

Reply:
Yup. As noted above ✌️ The speed-up initially measured seems to have come from somewhere else. I've dabbled with this a bit more yesterday and wasn't able to get any improvements even when using `--mode=development`. For now, I'll just update the PR to remove the flags (since they're the default anyway).

Reply:
Oh, sorry for the duplicate! Did not see your comment above 🙈